src/pynxtools_xps/kratos/__init__.py

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
```

Comment:
This is something I generally ask myself. Shall we really write nomad here or rather pynxtools or even FAIRmat (as this is not really part of nomad - I can be but must not be). Just leaving the comment here that we might want to have a short general discussion at some point.

Reply:
Yeah agreed. I guess it's fine for now, but we can discuss this later.